egs2/librimix/diar_enh1/conf/tuning/train_diar_enh_convtasnet_lmf_adapt.yaml

```diff
@@ -0,0 +1,102 @@
+optim: adam
```

Comment:
Can we try to simplify the config by combining some of them? The names and differences are not very clear to me now.

Reply:
I merged "lmf" files to the non-"lmf" files. I kept the "adapt" files since they are used when running `local/run_adapt.sh` for adaptation.

Reply:
Sure, will do!